src/raftstore/store/store.rs

```diff
@@ -335,7 +335,13 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         for region_id in self.region_ranges.values() {
             let region = self.region_peers[region_id].region();
             let start_key = keys::enc_start_key(region);
-            try!(delete_all_in_range(&self.engine, &last_start_key, &start_key));
+            if last_start_key < start_key {
+                info!("[store {}] cleanup [{}, {})",
+                      self.store_id(),
+                      escape(&last_start_key),
+                      escape(&start_key));
+                try!(delete_all_in_range(&self.engine, &last_start_key, &start_key));
```

Comment:
We are cleaning the garbage data here, but i didn't see any explanation of garbage data.